app/webhook.go

```diff
@@ -110,10 +110,15 @@ func (a *App) TriggerWebhook(payload *model.OutgoingWebhookPayload, hook *model.
 					l4g.Error(utils.T("api.post.handle_webhook_events_and_forget.event_post.error"), err.Error())
 				} else {
 					defer consumeAndClose(resp)
+
 					webhookResp := model.OutgoingWebhookResponseFromJson(resp.Body)
 
+					postRootId := ""
```

Comment:
Minor but can we move this inside the if block? Better to only allocate it when needed

Reply:
done